docs/installation/docker.md

```diff
@@ -327,4 +327,24 @@ services:
     # ...
 ```
 
+### Prevent in-container privilege escalation
+
+For additional security, you may [run Recyclarr with `--security-opt=no-new-privileges`][security_opt]
+in order to prevent privilege escalation. This will prevent the container from potentially gaining new
+privileges via `setuid` and `setgid` binaries, like `su` and `sudo`.
+To do this in Docker Compose, you need to add the `security_opt: ['no-new-privileges:true']` setting
+to your service configuration.
+
+Using the example `docker-compose.yml` presented at the start of this page, make the following
+modification:
```

Comment:
```suggestion
For additional security, you may [run Recyclarr with
`--security-opt=no-new-privileges`][security_opt] in order to prevent privilege escalation. This
will prevent the container from potentially gaining new privileges via `setuid` and `setgid`
binaries, like `su` and `sudo`.

To do this in Docker Compose, you need to add the `security_opt: ['no-new-privileges:true']` setting
to your service configuration. Using the example `docker-compose.yml` presented at the start of this
page, make the following modification:
```

I recommend this modification to 1) define the paragraphs a little clearer and 2) respect the existing max line length.